### ISSUE TITLE
Slice index must be integers error

### ISSUE THREAD

**Issue by aque** [jawatson/pythonprop#4]
I am running pythonprop 0.22 under OSX through MacPorts python3.4 and I am getting the following error:

```
Traceback (most recent call last):
  File "/opt/local/lib/python3.4/site-packages/pythonprop/voaP2PPlotgui.py", line 184, in run_plot
    parent = plot_parent)
  File "/opt/local/lib/python3.4/site-packages/pythonprop/voaP2PPlot.py", line 215, in __init__
    image_buffer = image_buffer[0:y_max-1,:]
TypeError: slice indices must be integers or None or have an __index__ method
```

**Comment by jawatson** [comment]
Many thanks for taking the time to report this and apologies for not getting round to fixing this earlier, I've only just notice the issue.  I believe that this is now fixed (4fca818)

**Comment by aque** [comment]
No problem. I verified this fixed the issue.
